examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -7060,9 +7094,9 @@ endpoint 1 {
     ram      attribute clusterRevision default = 1;
   }
 
-  server cluster LaundryDryerControls {
```

Comment:
Why is this turning off LaundryDryerControls?

Reply:
Bad merge, I assume. Reverted.